cfg/VideoStream.cfg

```diff
@@ -32,7 +32,7 @@ gen.add("exposure", double_t, LEVEL.RUNNING, "Target exposure", 0.5, 0.0, 1.0)
 gen.add("loop_videofile", bool_t, LEVEL.RUNNING, "Loop videofile", False)
 gen.add("reopen_on_read_failure", bool_t, LEVEL.RUNNING, "Re-open camera device on read failure", False)
 gen.add("output_encoding", str_t, LEVEL.NORMAL, "Output encoding", "bgr8")
-gen.add("start_frame", int_t, LEVEL.NORMAL, "Start frame of the video ", 0, 0)
-gen.add("stop_frame", int_t, LEVEL.NORMAL, "Stop frame of the video", -1, -1)
+gen.add("start_frame", int_t, LEVEL.RUNNING, "Start frame of the video ", 0, 0)
+gen.add("stop_frame", int_t, LEVEL.RUNNING, "Stop frame of the video", -1, -1)
```

Comment:
Incrased the level to force the dynamic_reconfigure callback to do `unsubscribe` and then `subscribe` so frame correctness checks are done.